src/main/java/com/logdyn/model/repositories/JiraRepository.java

```diff
@@ -107,7 +107,7 @@ else if (resCode == 401)
             }
         }
         catch (final IOException ioe){
-            LOGGER.debug(String.format("IOException when getting task '%s' from Jira Repository '%s'", id, this.name), ioe);
+            LOGGER.error(String.format("IOException when getting Task '%s' from Jira Repository '%s'", id, this.name), ioe);
```

Comment:
Probably keep as debug as these are going to caught further up the stack, but this is handy for tracing

Reply:
OK I'll revert as suggested - but its worth saying this logging is currently an anti pattern - see https://community.oracle.com/docs/DOC-983543, in particular the Antipatterns section halfway down "Log and throw".

Based on the article, we should either be just rethrowing or just logging, not both - the idea being the top level try/catch will be logging the Exception anyway. With this pattern, the Exceptions themselves should be appropriately named subclasses of Exception and should be initialised with an appropriate message.

That being said, for a project this size its probably not worth implementing exception handling properly. Up to you guys whether you want an issue raised to address this or not.